src/multicharge/param.f90

```diff
@@ -42,72 +43,69 @@ module multicharge_param
 
 contains
 
-subroutine new_eeq2019_model(mol, model, dielectric)
-   !> Molecular structure data
-   type(structure_type), intent(in) :: mol
-   !> Electronegativity equilibration model
-   class(mchrg_model_type), allocatable, intent(out) :: model
-   !> Dielectric constant of the medium
-   real(wp), intent(in), optional :: dielectric
-
-   real(wp), allocatable :: chi(:), eta(:), kcnchi(:), rad(:), rcov(:)
-   type(eeq_model), allocatable :: eeq
-
-   chi = get_eeq_chi(mol%num)
-   eta = get_eeq_eta(mol%num)
-   kcnchi = get_eeq_kcnchi(mol%num)
-   rad = get_eeq_rad(mol%num)
-   rcov = get_covalent_rad(mol%num)
-
-   allocate(eeq)
-   call new_eeq_model(eeq, mol=mol, chi=chi, rad=rad, eta=eta, kcnchi=kcnchi, &
-      & cutoff=25.0_wp, cn_exp=7.5_wp, rcov=rcov, cn_max=8.0_wp, & 
-      & dielectric=dielectric)
-   call move_alloc(eeq, model)
-
-end subroutine new_eeq2019_model
-
-subroutine new_eeqbc2024_model(mol, model, dielectric)
-   !> Molecular structure data
-   type(structure_type), intent(in) :: mol
-   !> Electronegativity equilibration model
-   class(mchrg_model_type), allocatable, intent(out) :: model
-   !> Dielectric constant of the medium
-   real(wp), intent(in), optional :: dielectric
-
-   real(wp), allocatable :: chi(:), eta(:), rad(:), kcnchi(:), &
-      & kqchi(:), kqeta(:), cap(:), rcov(:), avg_cn(:), en(:)
-   type(eeqbc_model), allocatable :: eeqbc
-
-   chi = get_eeqbc_chi(mol%num)
-   eta = get_eeqbc_eta(mol%num)
-   rad = get_eeqbc_rad(mol%num)
-   kcnchi = get_eeqbc_kcnchi(mol%num)
-   kqchi = get_eeqbc_kqchi(mol%num)
-   kqeta = get_eeqbc_kqeta(mol%num)
-   cap = get_eeqbc_cap(mol%num)
-   rcov = get_eeqbc_cov_radii(mol%num)
-   avg_cn = get_eeqbc_avg_cn(mol%num)
-   ! Electronegativities normalized to Fluorine 
-   ! with actinides (Th-Lr) set to average of 1.30 
-   en = get_pauling_en(mol%num)
-   en = merge(en, 1.30_wp, mol%num < 90)
-   en = merge(0.80_wp, en, mol%num == 87)
-   en = merge(1.00_wp, en, mol%num == 89)
-   en = merge(1.10_wp, en, mol%num == 90 .or. mol%num == 91 &
-      &.or. mol%num == 92 .or. mol%num == 95)
-   en = merge(1.20_wp, en, mol%num == 93 .or. mol%num == 94 &
-      &.or. mol%num == 97 .or. mol%num == 103)
```

Comment:
This shouldn't be overwritten. Those are the new parameters I added for the EN values. Please check my commit that all of the changes persist (also in the tests for the energies and charges).